lib/oauther.ex

```diff
@@ -94,7 +100,7 @@ defmodule OAuther do
   end
 
   defp nonce do
-    :crypto.rand_bytes(32) |> Base.encode64
+    :crypto.rand_bytes(16) |> Hexate.encode
```

Comment:
What is the reason for his change? RFC states that "nonce is a random string, uniquely generated", and what previous code does.